frontend/simple/js/state.js

```diff
@@ -24,7 +25,8 @@ const state = {
   currentGroupId: null,
   contracts: {}, // contractIds => type (for contracts we've successfully subscribed to)
   pending: [], // contractIds we've just published but haven't received back yet
-  loggedIn: false // TODO: properly handle this
+  loggedIn: false, // TODO: properly handle this
+  users: {} // contains a shortlist of a users commonly interacted with identity contracts
```

Comment:
It may even be worth renaming this variable from `users` to `usernameToContract`, or something more descriptive like that. Self-documenting code = good code that lasts.